test/integration/requests.test.ts

```diff
@@ -256,4 +256,98 @@ describe("/requests", function () {
         });
     });
   });
+
+  describe("GET /requests", function () {
+    it("should return all requests", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          done();
+        });
```

Comment:
Can you check received response have data or not?

Reply:
done.